tests/test_studies.py

```diff
@@ -0,0 +1,233 @@
+                  ]
+    },
+    {
+      "agent_key": "NT06aqHUT5djI1_JPAsck.data.inertia.1.1",
+      "sim_index": [ 
+                      {"fn": "=", "arg": "Baseline_dry_inertia_Y.Y"}, 
+                      { "fn_chain": [
+                          {"fn": "/", "arg": "Baseline_mass"}, 
+                          {"fn": "*", "arg": "SAR-10237_mass_delta"}, 
+                          {"fn": "+", "arg": "Baseline_dry_inertia_Y.Y" } ] 
+                      },
+                      { "fn_chain": [
+                          {"fn": "/", "arg": "Baseline_mass"}, 
+                          {"fn": "*", "arg": "SAR-10239_mass_delta"}, 
+                          {"fn": "+", "arg": "Baseline_dry_inertia_Y.Y" } ] 
+                      },
+                      { "fn_chain": [
+                          {"fn": "/", "arg": "Baseline_mass"}, 
+                          {"fn": "*", "arg": "P33081_mass_delta"}, 
+                          {"fn": "+", "arg": "Baseline_dry_inertia_Y.Y" } ] 
+                      },
+                  ]
+    },
+    {
+      "agent_key": "NT06aqHUT5djI1_JPAsck.data.inertia.2.2",
+      "sim_index": [ 
+                      {"fn": "=", "arg": "Baseline_dry_inertia_Z.Z"}, 
+                      { "fn_chain": [
+                          {"fn": "/", "arg": "Baseline_mass"}, 
+                          {"fn": "*", "arg": "SAR-10237_mass_delta"}, 
+                          {"fn": "+", "arg": "Baseline_dry_inertia_Z.Z" } ] 
+                      },
+                      { "fn_chain": [
+                          {"fn": "/", "arg": "Baseline_mass"}, 
+                          {"fn": "*", "arg": "SAR-10239_mass_delta"}, 
+                          {"fn": "+", "arg": "Baseline_dry_inertia_Z.Z" } ] 
+                      },
+                      { "fn_chain": [
+                          {"fn": "/", "arg": "Baseline_mass"}, 
+                          {"fn": "*", "arg": "P33081_mass_delta"}, 
+                          {"fn": "+", "arg": "Baseline_dry_inertia_Z.Z" } ] 
+                      },
+                  ]
+    },
+  ]
+}
+
+def getStudyStatus(scenario_branch_id, study_id):
+     study_control_resource = f'/simulations/branches/{scenario_branch_id}/control/study/{study_id}'
+     study_status = sedaroAPI.request.get(study_control_resource)
+     return study_status
+
+def getStudySimJobsStatus(scenario_branch_id, study_id):
+     study_status = getStudyStatus(scenario_branch_id, study_id)
+     study_job_ids = study_status['jobs']
+     return [ ( f"SimJob ID: {job['id']}", f"Status: {job['status']}", f"Progress:", job['progress']) 
+        for job_id in study_job_ids 
+        for job in [sedaroAPI.request.get(f'/simulations/branches/{scenario_branch_id}/control/{job_id}')] ]
+
+
+def runStudy(scenario_branch_id, iterations, overridesID):
+    create_study_resource_url = f'/simulations/branches/{scenario_branch_id}/control/study/'
+    new_studyjob = sedaroAPI.request.post(  create_study_resource_url,
```

Comment:
Do we not have methods for this in the client that were added as part of this effort? Hooks to start a study, poll its status, etc.?  I think we talked about this at the demo/review.  Let's discuss if its not implemented currently.